addons/csw/stringtable.xml

```diff
@@ -156,6 +165,7 @@
         </Key>
         <Key ID="STR_ACE_CSW_progressBarTimeCoefficent_displayName">
             <English>Interaction Time Coefficent</English>
+            <German>Interaktionszeit Koeffizient</German>
```

Comment:
```suggestion
            <German>Interaktionsdauerkoeffizient</German>
```
Wenn das zu donaudampfschifffahrtskapitänsmützig is, alternativ französisch: "Koeffizient für Interaktionsdauer"